CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+- Enable Main Thread Checker by default [#2549](https://github.com/tuist/tuist/pull/2549) by [@myihsan](https://github.com/myihsan)
```

Comment:
Just quick comment. Should it be under "Added" or "Changed"?

Reply:
Oh, sorry for the missing title.
Let me see if there is any other PR I can create and add the title in the meantime.